CMakeLists.txt

```diff
@@ -1,20 +1,25 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required(VERSION 3.10)
 
 project ("Xournal++" CXX C)
 
+## Project related Variables
 ## Also update changelog in debian folder!
 set (CPACK_PACKAGE_VERSION_MAJOR "1")
 set (CPACK_PACKAGE_VERSION_MINOR "0")
-set (CPACK_PACKAGE_VERSION_PATCH "13")
+set(CPACK_PACKAGE_VERSION_PATCH "19")
```

Comment:
Where does the 19 come from? We are still in 1.0.13 and will switch to 1.0.14 at release but this switch will be done in a separate commit.

Reply:
Its like 1.0.13 the development version of 1.1.0, instead of 1.0.14. And after we release 1.0.14 we will work on 1.1.

Reply:
Still this should be in its own commit. It is a version change and those should be very clear in the history. Additionally it should be done AFTER all commits of the version have been merged. This is so all commits can be sorted to one version without guessing.